test/types/main.test-d.ts

```diff
@@ -8,6 +9,8 @@ type Headers = BusboyHeaders;
 type Events = BusboyEvents;
 type BB = Busboy;
 
+new BusboyESM({ headers: { 'content-type': 'foo' } }); // $ExpectType Busboy
```

Comment:
you are using dtslint syntax for asserting types, while we use tsd. explicit `expectType` assertions should be used instead.

Reply:
done